src/services/birds.js

```diff
@@ -0,0 +1,10 @@
+export const postBird = (bird, season) => {
+    return fetch('http://localhost:7890/birds', {
+        method: 'POST',
+        headers: {
+            'Content-Type': 'application/json'
+        },
+        body: JSON.stringify({ bird, season })
+    })
+    .then(res => res.json());
+};
```

Comment:
files should end with a newline. you can configure vs code to add it for you automatically